kuma/wiki/management/commands/render_document.py

```diff
@@ -39,12 +40,7 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
         self.options = options
-
-        self.base_url = options['baseurl']
-        if not self.base_url:
-            from django.contrib.sites.models import Site
-            site = Site.objects.get_current()
-            self.base_url = 'http://%s' % site.domain
+        self.base_url = options['baseurl'] or absolutify('')
```

Comment:
I've left a comment for the absolutify helper, I think this shouldn't be `absolutify('')` but `absolutify('/')` to signal that it's the root URL path.